### DISCUSSION TITLE
Overleaf: Package biblatex Warning

### DISCUSSION THREAD

**Discussion by sibguerra** [joaomlourenco/novathesis#315] [Q&A]
How can we fix the following error (Overleaf)?

(template.tex 203)
Package biblatex Warning: Conflicting options. 'date=iso' requires 'seconds=true'. Setting 'seconds=true' on input line 203.

**Comment by joaomlourenco** [comment]
That's not an error. That's warning. It is safe to ignore it (this one in particular).

**Reply by sibguerra** [reply]
Thank you!
